res/values-vi/strings.xml

```diff
@@ -1230,6 +1309,16 @@
     <string name="Plasma_Boost_1_Day_">1 Ngày Tăng Plasma!</string>
     <string name="Plasma_Boost_2_Hours_">2 Tiếng Tăng Plasma.</string>
     <string name="Plasma_Boost">Tăng Plasma</string>
+    <string name="skin_warning" translatable="true">
+        "Your custom skin was not approved. Here are some of the common reasons for this. You should NOT upload:
+
+            1. A picture of someone shirtless;
+            2. A picture of someone in a suggestive pose or scene;
+            3. A picture that contains depictions of drugs (alcohol/cigar/weed);
+            4. An already existing Nebulous skin (such as level, achievement, season, plasma or exclusive skins from a pack). That skin must be modified in some meaningful way.
+
+        If you attempt on uploading the same refunded/rejected skin several times, you may receive more penalties or a ban."
+    </string>
```

Comment:
This one is not completely based on `res/values`; improved reading in untranslated.